grapher/scatterCharts/ScatterPlotChart.tsx

```diff
@@ -216,10 +216,14 @@ export class ScatterPlotChart
         return this.manager.table
     }
 
+    @computed private get transformedTableBeforeTimeFilterAndAvgAnnualChange() {
```

Comment:
I'm at a loss for new names for intermediary table :(

I also don't find the pattern to just save intermediary tables _great_.

Reply:
`WhatDoYouMeanIThinkThatLooksJustFine`

Reply:
It is an interesting issue – we are in general incentivised to create as many intermediate computed props as possible to minimise computation. It feels like there is some opportunity to create a better programming pattern/tool/language.